### PR TITLE
Bingle

### PR DESCRIPTION
**Pull request by BinglPringl**
What's new in next devlog?:

- Added drifter arm (don't mention. will be secret)

- Alchemist is now the only class that can craft medium and strong potions.

- Modded boards are now compatible with various modded recipes (Still WIP) 

- Bear head helmet now craftable with all bear pelt types

- Cotton, burlap, and woolen sacks now have 6 storage slots to match with linen sacks

- Fixed an issue where wool and cotton helmets couldn't be repaired

- Pets disobedience rate lowered by 10x for all pets

- Bullseye can now be toggled on and off for individual weapons

- Marjoram is now able to be cooked into fire-cured marjoram for shag.

- Pipeleaf plants now give xskills benefits

### CODE REVIEW

**Review by ShermansWorld — COMMENTED** [review]
Approved. looks good. Lots of little fixes

File: assets/alathraextras/itemtypes/tool/bullseye/bullseyedisabledbow.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
+{
+	code: "bow",
```
Comment (ShubiMaja) [inline assets/alathraextras/itemtypes/tool/bullseye/bullseyedisabledbow.json]: Let's please use proper json even though game allows json5

File: assets/alathraextras/itemtypes/tool/bullseye/bullseyedisabledbow.json [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+		},
+		"bow-long": {
+			translation: { x: 0, y: 0.5, z: 0 },
+			rotation: { x: 4, y: 1, z: 84 },
+			origin: { x: 0.5, y: 0, z: 0.5 },
+			scale: 3.41
+		},
+		"*": {
+			translation: { x: 0, y: 0.5, z: 0 },
+			rotation: { x: 4, y: 1, z: 84 },
```
Comment (ShubiMaja) [inline assets/alathraextras/itemtypes/tool/bullseye/bullseyedisabledbow.json]: Let's use a standard for indentation that does not involve tabs. i.e. 2 or 4 spaces for indents

Comment (ShubiMaja) [reply assets/alathraextras/itemtypes/tool/bullseye/bullseyedisabledbow.json]: @BinglPringl  most editors have a setting to choose indent. I think for vs this is what you need https://stackoverflow.com/questions/14167033/visual-studio-replace-tab-with-4-spaces

**Pull request merged**